rules/composer/src/ValueObject/ComposerModifier/AddPackageToRequire.php

```diff
@@ -20,23 +19,18 @@ final class AddPackageToRequire implements ComposerModifierInterface
     private $packageName;
 
     /**
-     * @var Version
+     * @var string
      */
     private $version;
 
-    /**
-     * @param string $packageName name of package (vendor/package)
-     * @param string $version target package version (1.2.3, ^1.2, ~1.2.3 etc.)
```

Comment:
Do you really want to remove this?

Reply:
Yea, these are obvious comments that do not any value.